platform/fakes/fake_platform.go

```diff
@@ -197,8 +199,14 @@ func (p *FakePlatform) SetupSSH(publicKey, username string) error {
 	return p.SetupSSHErr
 }
 
-func (p *FakePlatform) SetUserPassword(user, encryptedPwd string) (err error) {
-	p.UserPasswords[user] = encryptedPwd
+func (p *FakePlatform) SetUserPassword(user string, encryptedPwd string) (err error) {
+	if user == boshsettings.VCAPUsername {
+		p.UserPassword = encryptedPwd
+	}
```

Comment:
@mattcui, I am in Raleigh all of next week and will see face-to-face all the key Bluemix folks. I plan to raise this issue with them as well as others. Let's sync at scrum on this before I leave on Sunday.